dockerizeit/master/jenkins.properties

```diff
@@ -14,6 +14,13 @@ credentials {
     description = ""
     path = "/var/jenkins_home/.ssh/.password"
   }
+  smtp {
```

Comment:
Email notifications configuration doesn't support credentials, i.e. you can't pick credentials from the drop-down list as you would do that for Git plugin configuration for jobs. In this way reference to credentials is a bit misleading. Moreover, you are relying on this closure to be named SMTP in the script which implicit coupling that is better to avoid. What I mean here that it is fine to refer to closure names within one file since that will be an explicit dependency. When we refer to closure name somewhere else it would be like creating underwater rock that would hit you as soon as you change closure name or delete it since there is no way for user to anticipate this dependency. Do you see what I mean?
Please consider moving user id and path to password file to the actual smtp configuration.

You could do something like this

```
 smtp {
  enabled = false
  port = "25"
  host = "1.1.1.1"
  reply_to_address = "no-reply@yourcompany.com"
  authentication {
    enabled = true
    login = "user"
    passwordFile = "path to passoword file"
  }
 }
```